behave/features/environment.py

```diff
@@ -14,6 +21,10 @@ def before_all(context):
     context.artifacts_dir = ARTIFACTS_DIRECTORY
     # dir for report fox_admin_downloads
     context.download_dir = DOWNLOAD_DIRECTORY
+    # Boolean for axe finding a11y issues or not
+    context.a11y_running = False
+    if not context.config.userdata.get("a11y", False):
+        context.config.userdata.update({"a11y": "false"})
```

Comment:
```    if not context.config.userdata.get("a11y", False):
        context.config.userdata.update({"a11y": "false"})```
context.config.userdata is a dict and dicts have a .setdefault method which lets you set an item in the dict if it's not already set https://docs.python.org/3.4/library/stdtypes.html?highlight=dict.setdefault#dict.setdefault

Reply:
Also maybe think about converting "a11y" to a boolean so that you don't have to keep comparing to a string

Reply:
Done, thanks